### PR TITLE
Feat/settings page

### PR DESCRIPTION
**Pull request by Thuhaa**
Thank you for submitting a pull request!

## Description

Please describe what you changed briefly.

### Type of Pull Request

- [x] Adding a feature
- [ ] Fixing a bug
- [ ] Maintaining documents
- [ ] Others ()

### Verify the followings

- [x] Code is up-to-date with the `develop` branch
- [x] No build errors after `pnpm build`
- [x] No lint errors after `pnpm lint`
- [x] Run the tests with pnpm test and lint the project with pnpm lint and pnpm check
- [x] Make sure all the existing features working well

### Changesets

- [ ] If your PR makes a change under `packages` folder that should be noted in one or more packages' changelogs, generate a changeset by running `pnpm changeset` and following the prompts. Changesets that add features should be `minor` and those that fix bugs should be `patch`. Please prefix changeset messages with `feat:`, `fix:`, or `chore:`.

Refer to [CONTRIBUTING.MD](https://github.com/UNDP-Data/geohub/blob/develop/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: sites/geohub/src/components/UserAccount.svelte [hunk 1]
```diff
@@ -15,6 +15,10 @@
   const handleDropdown = () => {
     dropdownActive = !dropdownActive
   }
+
+  const openSettingsPage = () => {
+    window.location.href = '/settings'
```
Comment (JinIgarashi) [inline sites/geohub/src/components/UserAccount.svelte:20]: can we use `goto` method provided by sveltekit?

https://kit.svelte.dev/docs/modules#$app-navigation-goto

File: sites/geohub/src/components/Settings.svelte [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+          on:click={() => (settings.sideBarPosition = 'left')}
+          class="column sidebar-col {settings.sideBarPosition === 'left' ? 'selected-sidebar' : ''}">
+          <div class="card">
+            <div class="card-header">
+              <p class="card-header-title">Left Sidebar</p>
+            </div>
+            <div class="card-content">
+              <div class="content">
+                <img
+                  src="/assets/sidebar/left-sidebar.png"
```
Comment (JinIgarashi) [inline sites/geohub/src/components/Settings.svelte:94]: I am wondering whether we can use this maximise icon for sidebar position.

https://fontawesome.com/icons/window-maximize?s=regular&f=classic

This screenshots also look good. but I feel the images are too large. I think smaller image is much better. maybe one-third of current image size.

**Review by JinIgarashi — CHANGES_REQUESTED** [review]
There are several bugs before you can merge it to develop branch.

Please select default position when we move to setting page first time.

image

Can't scroll anymore for both data tab and layer tab.

image

In responsive, close button is shown in weird position. it should be shown in top-right.
image


Is it possible to show some message after clicking `apply` button? Maybe we can use `Notification` component.

**Comment by Thuhaa** [comment]
**Review Request Fixes**
1. Smaller Image sizes and addition of icons to represent right and left side of sidebar
image

2. Notification of `Applied Settings`
image

3. Scroll fix
image

4. `Close` icon position in sidebar
image

File: sites/geohub/src/components/App.svelte [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  const getCookie = (name: string) => {
+    const value = `; ${document.cookie}`
+    const parts = value.split(`; ${name}=`)
+    if (parts.length === 2) return parts.pop()?.split(';').shift()
+  }
+
+  let sideBarPosition: 'left' | 'right' = JSON.parse(getCookie('settings') || '{}').sideBarPosition || 'left'
+  $: isMobile = innerWidth < 768
+  $: splitHeight = innerHeight - headerHeight
+  $: console.log(document.cookie)
```
Comment (JinIgarashi) [inline sites/geohub/src/components/App.svelte:29]: delete console.log

File: sites/geohub/src/components/Content.svelte [hunk 4]
```diff
@@ -2,8 +2,9 @@
   import DataView from '$components/DataView.svelte'
   import LayerList from '$components/LayerList.svelte'
   import { TabNames } from '$lib/constants'
-  import { layerList } from '$stores'
-  import { Tabs, type Tab } from '@undp-data/svelte-undp-design'
+  import { layerList, map } from '$stores'
+  import { Tabs } from '@undp-data/svelte-undp-design'
+  import type { Tab } from '@undp-data/svelte-undp-design/dist/interfaces'
```
Comment (JinIgarashi) [inline sites/geohub/src/components/Content.svelte:7]: please delete `/dist/interface`

File: sites/geohub/src/components/DataView.svelte [hunk 5]
```diff
@@ -323,8 +323,8 @@
 </div>
 
 <style lang="scss">
-  .data-view-container {
-    overflow-y: auto;
+  :global(.data-view-container) {
+    overflow-y: auto !important;
```
Comment (JinIgarashi) [inline sites/geohub/src/components/DataView.svelte:327]: did you confirm scrolling in layer tab?

**Review by JinIgarashi — CHANGES_REQUESTED** [review]
I commented two things

File: sites/geohub/src/components/Content.svelte [hunk 6]
```diff
@@ -4,7 +4,7 @@
   import { TabNames } from '$lib/constants'
   import { layerList, map } from '$stores'
   import { Tabs } from '@undp-data/svelte-undp-design'
-  import type { Tab } from '@undp-data/svelte-undp-design/dist/interfaces'
+  import type { Tab } from '@undp-data/svelte-undp-design'
```
Comment (JinIgarashi) [inline sites/geohub/src/components/Content.svelte:7]: can you merge line no. 6 and no.7 to a line?

**Review by JinIgarashi — CHANGES_REQUESTED** [review]
please do just a small refactoring which I commented

**Review by JinIgarashi — APPROVED** [review]

**Pull request merged**